cer-graphql/tests/integration-tests.test.js

```diff
@@ -76,6 +138,12 @@ describe('Authorisation resolvers', () => {
         expect(res.errors[0].extensions.code).toEqual('UNAUTHENTICATED');
     });
 
+    test('Requesting an articleCollection non-public field with a valid Authorization header returns an response', async function () {
+        let { query } = await createServerAndTestClientWithAuth();
+        let res = await query({ query: TQ.GET_ARTICLE_COLLECTION_PRIVATE_WITH_SSO });
+        expect(res.data.articleCollection).toBeTruthy();
+    }, 20000);
```

Comment:
Nice! But I'd suggest defining a `TIMEOUT_PERIOD` const at the top of this file in case we want to do this sort of thing elsewhere, just like I've done in the web's `e2e` test file.